gms/impl/src/main/java/com/linkedin/metadata/configs/GlossaryNodeSearchConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.metadata.configs;
+
+import com.linkedin.metadata.dao.search.BaseSearchConfig;
+import com.linkedin.metadata.dao.utils.SearchUtils;
+import com.linkedin.metadata.search.GlossaryNodeInfoDocument;
+
+import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.Set;
+
+
+public class GlossaryNodeSearchConfig extends BaseSearchConfig<GlossaryNodeInfoDocument> {
+  @Override
+  @Nonnull
+  public Set<String> getFacetFields() {
+    return Collections.emptySet();
+  }
+
+  @Override
+  @Nonnull
+  public Class<GlossaryNodeInfoDocument> getSearchDocument() {
+    return GlossaryNodeInfoDocument.class;
+  }
+
+  @Override
+  @Nonnull
+  public String getDefaultAutocompleteField() {
+    return "definition";
```

Comment:
Do glossary node documents not have a `name` field we could autocomplete on?

Reply:
As of now we wanted to put on definition field, we will take a look again when we are building UI